svthalia/concrexit
# Fix alumni mailinglist missing people

Closes #3435.



### Summary
This changes the way the oldmembers mailinglist is determined to include alumni from before concrexit, of whom we often have no (accurate) membership history.

I branched off of the branch for #3420, as both will probably be in the same release, and it would need a harder rebase otherwise.


### How to test

1. Make a new member
2. Check that they can set, unset and reset the receive_oldmembers setting.
3. Make a new benefactor
4. Check that they cannot set the receive_oldmembers setting to True.
